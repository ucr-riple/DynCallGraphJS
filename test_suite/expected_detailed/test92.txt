{
    'global (test92.js:1:1:7:5)': [
        'anon (test92.js:1:2:7:2)'
    ],
    'anon (test92.js:5:15:5:20)': [
        'foo (test92.js:2:3:4:4)'
    ],
    'anon (test92.js:6:3:6:13)': [
        'anon (test92.js:3:13:3:27)'
    ]
}