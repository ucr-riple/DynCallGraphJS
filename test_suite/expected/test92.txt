{ '(test92.js:1:1:7:5)':
   [ '(test92.js:1:2:7:2)' ],
  '(test92.js:5:15:5:20)':
   [ '(test92.js:2:3:4:4)' ],
  '(test92.js:6:3:6:13)':
   [ '(test92.js:3:13:3:27)' ] }